bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/charge-channel-types.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mercedesme"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="charge-flap-channel">
+		<item-type>Number</item-type>
+		<label>Charge Flap</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Open</option>
+				<option value="1">Closed</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-ac-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler AC</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Plugged</option>
+				<option value="2">Unplugged</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-dc-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler DC</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Plugged</option>
+				<option value="2">Unplugged</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="coupler-lock-channel">
+		<item-type>Number</item-type>
+		<label>Charge Coupler Lock</label>
+		<state readOnly="true">
+			<options>
+				<option value="0">Locked</option>
+				<option value="1">Unlocked</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="active-channel">
+		<item-type>Switch</item-type>
+		<label>Charge Active</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="power-channel">
+		<item-type>Number:Power</item-type>
+		<label>Charge Power</label>
+		<state pattern="%.1f kW" readOnly="true"/>
+	</channel-type>
+	<channel-type id="program-channel">
+		<item-type>Number</item-type>
+		<label>Charge Program</label>
```

Comment:
It's described in [readme section](https://github.com/weymann/MercedesMe-OH-Drops?tab=readme-ov-file#program-mapping).
It can differ between cars and only the programs which are available are shown as options (Vehicle Capabilities). E.g. hybrids (seems) to have only one program while pure EVs have more.